scripts/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+"""
+Kotlin-based scripts to help developers or perform continuous integration tasks.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_binary", "kt_jvm_test")
+load("//scripts/src/java/org/oppia/android/scripts/regex:regex_pattern_assets.bzl", "retrieve_regex_pattern_check_assets")
+
+REGEX_PATTERN_CHECK_ASSETS = retrieve_regex_pattern_check_assets("//scripts/org/oppia/android/scripts/regex/assets")
+
+# Visibility for libraries that have binaries defined at this package, and libraries defined at
+# subpackage level.
+package_group(
+    name = "oppia_script_lib_visibility",
+    packages = [
+        "//scripts",
+        "//scripts/src/java/org/oppia/android/scripts/...",
```

Comment:
@BenHenning 
The sole reason for adding this was because many of the subpackage libraries need other subpackage libraries as a dep (for example: libraries of `common` package are needed by `regex_pattern_validation_check_lib`), so in order to make them visible to each other, we probably have to include the sub-packages too in this package group.
Does this sounds good?

Reply:
We should have two different package groups for this purpose, but I think you can wait for my PR at this point since you need it to pass CI on this PR, anyway, and it sets up a lot of this by building on the approach you've established here.